src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/Client/_Imports.razor

```diff
@@ -0,0 +1,11 @@
+﻿@using System.Net.Http
+@using System.Net.Http.Json
+@using Microsoft.AspNetCore.Components.Routing
+@using Microsoft.AspNetCore.Components.Web
+@using Microsoft.AspNetCore.Components.WebAssembly.Http
+@using Microsoft.JSInterop
+@*#if (!Hosted)
+@using EmptyComponentsWebAssembly_CSharp
+#else
+@using EmptyComponentsWebAssembly_CSharp.Client
+#endif*@
```

Comment:
Just confirming this is intentionally commented out, because that's how Razor's templating works? Is this how it is in the regular template?

Reply:
Yes, this is the formatting from the default template.